package.json

```diff
@@ -9,17 +9,19 @@
     }
   },
   "dependencies": {
+    "lodash": "~2.4.1",
+    "q": "~0.9.7",
+    "vumigo_v01": "~0.1.22",
+    "vumigo_v02": "~0.2"
+  },
+  "optionalDependencies": {
```

Comment:
The only alternatives I can think of are listing them in a file and running `cat extra-node-deps.txt | xargs npm install` or something, or having a second `package.json` with the extra deps it in that we can install separately. I really don't want to leave these in `dependencies`, because installing `libxmljs` takes rather a long time.

(FWIW, these deps *are* optional in the sense that nothing in vumi-go depends on them -- they're only required by some jsbox apps.)